CHANGELOG.md

```diff
@@ -1,23 +1,28 @@
 # Changelog
 
+## 4.7.0
+* [ADDED] AuthenticateUser
+* [ADDED] AuthorizeChannel
+* [DEPRECATED] Authenticate
+
```

Comment:
add that you bumped `Newtonsoft.Json` version

Reply:
Sure